doc-downloader.functions.ps1

```diff
@@ -25,8 +31,14 @@ function CheckDirectory
 function  MakeHttpGetCall
 {param([string]$URI, [hashtable]$HEADERS, [string]$LOG_FILE)
     $ProgressPreference = 'SilentlyContinue'
-    $response = Invoke-WebRequest -Uri $URI -Headers $HEADERS -MaximumRedirection 0 -ErrorAction SilentlyContinue -ErrorVariable $ProcessError
-    #$ProgressPreference = 'Continue'   
+    $powershellVersion = (Get-Host).Version.Major
+    if($powershellVersion -eq 7){
```

Comment:
>=? What about when 8 comes out?

Reply:
@pnc Yeah I was thinking this. whats the best way to go about future proofing this? it seems that we just have to make changes as powershell gets changed yeah?

Reply:
I think greater than or equal to is fine for now. They're unlikely to _remove_ this feature when 8 drops. We may need to do _additional_ fixes then.

Reply:
ah yeah! good point, thanks!